showyourwork/workflow/build.smk

```diff
@@ -18,44 +22,39 @@ workdir: paths.user().repo.as_posix()
 # What kind of run is this? (clean, build, etc.)
 run_type = get_run_type()
 
+if not (paths.user().temp / "config.json").exists() and run_type != "clean":
+    raise exceptions.MissingConfigFile()
```

Comment:
I just made a second if statement instead of an else. But I'm not sure if `onsuccess` refers to the original if statement or not?

Also - with this exception raised at the top, does the rest even need to be embedded in an if statement?